### PR TITLE
web-angular: support running component tests in the browser

### LINKED ISSUE #445: web-angular: add `test/app_test.html` in support of in-browser test monitoring
Hitting 8081 for the generated test fails.

>[test] GET  → Could not find asset meetme|test.

### PR DESCRIPTION
**Pull request by chalin**
Fixes #445

I've tested this and it seem to work. I.e., the framework reports that test pass, even if a "Bad state" exception is thrown (https://github.com/dart-lang/angular/issues/631).

### CODE REVIEW

File: templates/web-angular/test/app_test.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
```
Comment (kevmoo) [inline templates/web-angular/test/app_test.html]: This file isn't needed – unless you're doing something special.

pkg/test will generate it automatically...

...unless I'm missing something

Comment (chalin) [reply templates/web-angular/test/app_test.html]: W/o this file I can't get the `angular_test`-based tests to run (I'm not saying that it is impossible, just that I don't know how).

I haven't run tests in this manner very much. I based the assumption that such a file was needed by looking at @matanlurey's [angular_test demo app](https://github.com/dart-lang/site-webdev/tree/master/examples/ng_test/github_issues). In particular see the [test](https://github.com/dart-lang/site-webdev/tree/master/examples/ng_test/github_issues/test) folder.

I'll let @matanlurey comment.

cc @kwalrath

Comment (kevmoo) [reply templates/web-angular/test/app_test.html]: Ah...maybe it's angular-test specific.

...ah! Or it's pub-serve specific. That's it, I bet!

Comment (chalin) [reply templates/web-angular/test/app_test.html]: Ok. @matanlurey?

**Comment by kevmoo** [comment]
@matanlurey ?

**Comment by kwalrath** [comment]
What's the status of this PR, @chalin? Are we still waiting for feedback from @matanlurey? If so, I can bug him...

**Comment by matanlurey** [comment]
We can't even run our own tests yet until @alorenzen and friends get it working.

So the answer is... TBD.

**Comment by chalin** [comment]
Closing this PR. We'll address this feature under Angular 5.

**Pull request closed without merging**